webui/src/layout/Sidebar/Sidebar.tsx

```diff
@@ -1,19 +1,22 @@
-// components/Sidebar.tsx
-import React from "react";
-import { Link } from "react-router-dom";
-import styles from "./Sidebar.module.scss";
-import Button from "./Button/Button";
-import { ReactComponent as BacalhauLogo } from "../../images/bacalhau.svg";
-import { ReactComponent as JobsIcon } from "../../images/jobs-icon.svg";
-import { ReactComponent as NodesIcon } from "../../images/nodes-icon.svg";
-import { ReactComponent as Cogwheel } from "../../images/cogwheel.svg";
+// layout/Sidebar/Sidebar.tsx
+import React from "react"
+import { Link } from "react-router-dom"
+import { ReactComponent as BacalhauIcon } from "../../images/bacalhau.svg"
+import { ReactComponent as JobsIcon } from "../../images/jobs-icon.svg"
+import { ReactComponent as NodesIcon } from "../../images/nodes-icon.svg"
+import { ReactComponent as CogwheelIcon } from "../../images/cogwheel.svg"
```

Comment:
Here is where you are using the other approach.

Reply:
See above